src/org/zaproxy/zap/extension/websocket/resources/Messages.properties

```diff
@@ -43,7 +43,7 @@ websocket.manual_send.fail.invalid_opcode       = Invalid WebSocket opcode selec
 websocket.manual_send.menu                      = Manual Send WebSocket Message
 websocket.manual_send.menu.mnemonic				= w
 websocket.manual_send.popup                     = Resend WebSocket Message
```

Comment:
Actually we may want to keep the "Message" (instead of "Request") these dialogues allow to send messages to both sides (client and server).